libs/sdk-ui-kit/src/Icon/Icon.tsx

```diff
@@ -28,9 +28,12 @@ export const Icon: React.FC<IIconProps> = (props) => {
         case "Undo":
         case "Home":
         case "BurgerMenu":
-        case "Rows": {
+        case "Rows":
+        case "DragHandle": {
             const IconComponent = Icons[name];
-            return <IconComponent name={name} color={color} colorPalette={colorPalette} />;
+            return (
+                <IconComponent className={className} name={name} color={color} colorPalette={colorPalette} />
```

Comment:
Maybe we could also introduce something like `classNames(className, "s-{name}.toLowerCase()")`. What do you think?

Reply:
Do you think we might need to test the icons? I am thinking about existence of an icon, but still not convinced we should add the `s-` class just for this very occasional use case.

Reply:
We decided to omit `s-` class name for now.